packages/israel/israel.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created for MapSVG plugin: http://mapsvg.com -->
+<svg
+   xmlns:mapsvg="http://mapsvg.com"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   mapsvg:geoViewBox="34.228663 33.434207 35.935383 29.496766"
+   width="294.62534"
+   height="792.60406">
```

Comment:
Could you remove the unnecessary attributes please? You can look at other SVG maps, `xmlns`, `viewBox` and `aria-label` attributes are enough.

If you are not sure how to do it, let me know and I will fix it before publishing the package.